tests/dns/dns-rcode/test.rules

```diff
@@ -0,0 +1,8 @@
+# Should alert in both directions as no flow is provided.
+alert dns any any -> any any (dns.rcode; content:"oisf"; sid:1; rev:1;)
```

Comment:
should it not be something like `dns.rcode:12` ?

Reply:
and no content

Reply:
Since it's an integer keyword ;)